src/components/global/BackButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function BackButton(props) {
+  const {buttonName} = props;
+  return (
+    <button type='button' className='btn'>
+      <i className='material-icons'>remove_circle</i>
```

Comment:
```suggestion
      <i className='material-icons'>arrow_back</i>
```

Reply:
But this icon matches the edit button icon nicely